cypress/e2e/settings/data-import/generated-376006.cy.js

```diff
@@ -0,0 +1,185 @@
+import { Permissions } from '../../../support/dictionary';
```

Comment:
I think the name of this file should be another

Reply:
It's short cut import of permissions.js modelu from '../../../support/dictionary/index.js'